mmv1/third_party/terraform/services/storage/data_source_storage_bucket_object_content.go

```diff
@@ -30,6 +32,22 @@ func DataSourceGoogleStorageBucketObjectContent() *schema.Resource {
 		Required:    false,
 	}
 
+	dsSchema["output_sha512"] = &schema.Schema{
+		Type:        schema.TypeString,
+		Description: "SHA512 checksum of object content.",
+		Computed:    true,
+		Optional:    false,
+		Required:    false,
+	}
+
+	dsSchema["output_base64sha512"] = &schema.Schema{
```

Comment:
generally I'd expect these fields to use `content_` as the prefix instead of `output_`. Is there a particular reason you're not doing that?